### DISCUSSION TITLE
关闭防火墙后过一段时间会自动开启，防火墙不能对UDP端口不能放行

### DISCUSSION THREAD

**Discussion by ErkundenSie** [midoks/mdserver-web#488] [Q&A] [answered]
如题，建议优化

**Comment by midoks** [comment, marked as answer]
1.关闭防火墙后过一段时间会自动开启。 
这个问题，需要我亲自验证，我还没有遇到过，可能是其他问题引起的。

2.防火墙不能对UDP端口不能放行 。
如果第一问题解决了，就不应该出现。

可能你系统的出现问题，你可使用其他服务测试一下这个问题。
通过命令行检查一下。

debian  ufw。
redhat  firewalld。

**Comment by ErkundenSie** [comment]
对于第二个，自带的防火墙我已经开了专门的UDP端口，在面板也开通了端口，但是UDP端口用不了，我把面板的防火墙关闭后，就能用了，应该是面板的问题

**Reply by ErkundenSie** [reply]
用的是最新的版本

**Reply by midoks** [reply]
已用
